config/settings/development.yml

```diff
@@ -1,3 +1,5 @@
 ---
 :log:
   :level_rails: debug
+:webservices:
+  :url: http://localhost:4000
```

Comment:
This looks good, but we need an empty entry in settings.yml for the new `url` key.

Reply:
```yaml
:webservices:
  :url:
```